packages/client/package.json

```diff
@@ -79,21 +79,18 @@
     "graphql-typed": "^0.6.1",
     "hoist-non-react-statics": "^3.3.0",
     "immutable": "3.8.2",
-    "intersection-observer": "^0.7.0",
```

Comment:
I don't think we need to polyfill this anymore. It's pretty well-supported: https://caniuse.com/intersectionobserver